pkg/ip/ip.go

```diff
@@ -26,6 +27,15 @@ const (
 	ipv6BitLen = 8 * net.IPv6len
 )
 
+var (
```

Comment:
I prefer byte arrays because they are explicit and are not changed in the background - Go's `net` library manipulates the byte arrays you provide to `net.IP` and `net.Mask`.  Using byte arrays is what is done in the `net` package as well. See L55: https://golang.org/src/net/ip.go .

Reply:
Fair enough.